pkg/utils/kubeutils.go

```diff
@@ -48,6 +49,11 @@ func BoolToCondition(val bool) metav1.ConditionStatus {
 	return metav1.ConditionFalse
 }
 
+// ConditionToBool converts boolean value to condition status.
+func ConditionToBool(status metav1.ConditionStatus) bool {
+	return status == metav1.ConditionTrue
```

Comment:
How many places do you use this function?

Reply:
2 places